packages/manager/modules/pci/src/projects/project/project.constants.js

```diff
@@ -172,6 +172,12 @@ export const LEGACY_PLAN_CODES = ['project.legacy', 'project.2018'];
 
 export const LOCAL_ZONE_REGION = 'localzone';
 
+export const URL_INFO = {
+  PREFIX: 'https://ovhcloud.com',
```

Comment:
**question:** Is it always the same for all regions?

Reply:
Will check this with Mounir as of now in the list the prefix is same.